package.json

```diff
@@ -16,6 +16,7 @@
     "build:performance": "yarn run build:performance:components && yarn run build:performance:benchmarks",
     "build:performance:components": "lerna exec --scope perf-benchmarks-components -- yarn build",
     "build:performance:benchmarks": "lerna exec --scope perf-benchmarks -- yarn build",
+    "dev": "lerna run dev --parallel --ignore perf-benchmarks --ignore perf-benchmarks-components --ignore integration-tests",
```

Comment:
`lerna --parallel` is not the perfect option for this kind of thing, but it looks like lerna does not plan on implementing a watch mode (https://github.com/lerna/lerna/issues/1722#issuecomment-429465600).

The downside is that you must run `yarn build` at least once before running `yarn dev`. Also it seems that very occasionally `yarn dev` may fail because a dependent package cannot find the `dist` files from a dependency (I've seen this happen once out of ~10 times).

There are various ["lerna watch"](https://www.npmjs.com/search?q=lerna%20watch) utilities on npm, but I don't know if it's worth it to add an extra dependency.

Reply:
Would it be worth prepending `lerna run build &&` to `dev`? Or if you want the option, rename what you have to `dev-no-build` and add `"dev": "lerna run build && lerna run dev-no-build"`.

Reply:
I considered that, but decided it against it because 1) `yarn` already runs `build`, and 2) if `yarn dev` fails because you forgot to `build`, then you find out almost immediately. It should be very rare for `yarn dev` to fail.